packages/mask/src/plugins/PoolTogether/base.tsx

```diff
@@ -3,9 +3,9 @@ import { POOLTOGETHER_PLUGIN_ID, URL_PATTERN } from './constants'
 
 export const base: Plugin.Shared.Definition = {
     ID: POOLTOGETHER_PLUGIN_ID,
-    name: { fallback: 'PoolTogether' },
+    name: { fallback: 'Pooltogether' },
     description: { fallback: 'PoolTogether is a protocol for no-loss prize games on the Ethereum blockchain' },
-    publisher: { name: { fallback: 'iRhonin' }, link: 'https://github.com/iRhonin' },
+    publisher: { name: { fallback: 'Mask Network' }, link: 'https://mask.io/' },
```

Comment:
?

Reply:
Please revert this change, we can't change publisher info!